### PR TITLE
doc: add tarantoolctl/cartridge to tt transition notes

### PR DESCRIPTION
**Pull request by psergee**
Part of #286

### CODE REVIEW

File: doc/examples.rst [hunk 1]
```diff
@@ -387,3 +387,126 @@ Flag without argument:
...
+    modules:
+        directory: modules
+    app:
+        run_dir: ./run/tarantool
+        log_dir: ./log/tarantool
+        log_maxsize: 100
+        log_maxage: 8
+        log_maxbackups: 10
+        restart_on_failure: false
+        data_dir: ./lib/tarantool
```
Comment (LeonidVas) [inline doc/examples.rst:429]: This parameter has been split.

Comment (LeonidVas) [reply doc/examples.rst:429]: The `tarantoolctl_layout` parameter is now also added.

File: doc/examples.rst [hunk 2]
```diff
@@ -387,3 +387,126 @@ Flag without argument:
    http://127.0.0.1:1024/tarantool
    {"res":"Tarantool CLI version 0.1.0, darwin/amd64. commit: bf83f33\n
     • Tarantool executable found: '/usr/local/bin/tarantool'\n"}
+
+Transition from tarantoolctl to tt
+----------------------------------
+Suppose we have a set of Tarantool instances managed by ``tarantoolctl`` utility. In order for the tt
```
Comment (LeonidVas) [inline doc/examples.rst:393]: I think we should also describe the case where the user does not have a `.tarantoolctl' file but uses `tarantoolctl' to run tarantool instances on the system.

**Review by LeonidVas — CHANGES_REQUESTED** [review]
Hi! Thank you for the patch.
See several comments bellow.

File: doc/examples.rst (hunk 2 above)
Comment (psergee) [reply doc/examples.rst]: Added a section for system-wide config.

File: doc/examples.rst (hunk 1 above)
Comment (psergee) [reply doc/examples.rst]: Updated.

**Review by LeonidVas — APPROVED** [review]

**Pull request merged**